Lib/configparser.py

```diff
@@ -562,35 +561,36 @@ def __init__(self):
 
 
 class _Line(str):
+    __slots__ = 'clean', 'has_comments'
 
     def __new__(cls, val, *args, **kwargs):
```

Comment:
Out of curiosity, why do we need an explicit `__new__`?

Reply:
Without it, the superclass gets called with `str.__new__(val, encoding=comments)`.

Reply:
Oh you're right.